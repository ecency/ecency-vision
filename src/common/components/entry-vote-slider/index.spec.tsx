import React from "react";
import VotingSlider from "./index";
import renderer from "react-test-renderer";

const props = {
  value: 100,
  mode: "up",
  setVoteValue: () => {}
};

const component = renderer.create(<VotingSlider {...props} />);

it("(1) Default render", () => {
  expect(component.toJSON()).toMatchSnapshot();
});
