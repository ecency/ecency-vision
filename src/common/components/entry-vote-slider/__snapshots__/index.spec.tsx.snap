// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(1) Default render 1`] = `
<div
  style={
    Object {
      "height": "40px",
      "paddingRight": "13px",
    }
  }
>
  <div
    className="slide"
    onMouseDown={[Function]}
    onMouseUp={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
  >
    <div
      className="slide-background-line"
    >
      <div
        className="slider-line"
        style={
          Object {
            "background": "#357ce6",
            "width": "100%",
            "zIndex": 100,
          }
        }
      />
      <span
        className="slider-dot pointer-dot"
        style={
          Object {
            "background": "#357ce6",
            "left": "100%",
          }
        }
      />
      <div
        className="dots-container"
      >
        <div
          style={
            Object {
              "width": "25%",
            }
          }
        />
        <div
          style={
            Object {
              "width": "25%",
            }
          }
        >
          <span
            className="slider-dot"
            style={
              Object {
                "background": "#357ce6",
              }
            }
          />
        </div>
        <div
          style={
            Object {
              "width": "25%",
            }
          }
        >
          <span
            className="slider-dot"
            style={
              Object {
                "background": "#357ce6",
              }
            }
          />
        </div>
        <div
          style={
            Object {
              "width": "25%",
            }
          }
        >
          <span
            className="slider-dot"
            style={
              Object {
                "background": "#357ce6",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
